vbuild/abuild.py

```
@@ -152,7 +154,7 @@ def abuild(
                 try:
                     container.stop()  # pyright: ignore[reportUnknownMemberType]
 
-                except Exception:
+                except Exception:  # noqa: S110
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)